packages/as-transformer/src/transformers/massaExportImports.ts

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import {
+  IdentifierExpression,
+  ImportDeclaration,
+  ImportStatement,
+} from 'assemblyscript/dist/assemblyscript.js';
+import { Update, GlobalUpdates } from './interfaces/Update.js';
+// import { IExpressionTransformer } from './interfaces/IExpressionTransformer';
+
+/**
+ * The MassaExportCalls is a specific transformer that works by replacing a any massa exported function calls
+ * by their transformed version.
+ */
+export class MassaExportImports {
+  isMatching(expression: string): boolean {
+    const calls = GlobalUpdates.get()
+      .filter((update: Update) => update.from === 'MassaExport')
+      .map((update: Update) =>
+        update.data.get('funcToPrivate')
+          ? update.data.get('funcToPrivate')![0]
+          : '',
+      );
+    return calls.includes(expression);
+  }
+
+  /**
+   * This method takes a {@link ImportDeclaration} and transforms it by
+   * replacing the node containing a call to the massa exported function found
+   * with an underscored version.
```

Comment:
Should be more straightforward, it's kind of hard to read.
Maybe you can only keep the last part beginning by "replacing" and put the rest in remarks if really needed.

Reply:
On second thoughts, i'll do the documentation of this class in the appropriate issue, since this class is not used at the moment. If it's ok ofc.